custom_types.yaml

```diff
@@ -587,12 +587,12 @@ node_types:
     interfaces:
       Standard:
         configure:
-          implementation: mesos/mesos_master_install.yml
+          implementation: artifacts/mesos/mesos_master_install.yml
```

Comment:
@MatMaul in other recipes we have used the full url. Please change it to use it:
https://raw.githubusercontent.com/indigo-dc/tosca-types/master/artifacts/mesos/mesos_master_install.yml